dashboards/individual-flows-per-country.json

```diff
@@ -380,9 +380,9 @@
       "type": "netsagenavigation"
     },
     {
-      "content": "<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>\n<h1><center><b>Individual Flows per Country</b></center></h1>\n<center>This dashboard shows per-flow level data, given a particular source and destination organization. \nThe table shows the volume, rate, retransmits, RTT and duration for each flow. </center>\n<center>The heatmaps below show volume, rate and retransmits of flows for the given source and destination organizations over a specific period of time.</center>\n<center>The horizontal axis represents days and the vertical axis represents time of day. Darker squares show higher data transmissions at those hours.</center>\n<center><b>Please note that retransmit and RTT information is only available for archive flow data, and a maximum of 10,000 results are shown in the tables.</b></center><center>All times are displayed in browser local time.</center>\n",
+      "content": "<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>\n<h1><center><b>Individual Flows per Country</b></center></h1>\n<center>This dashboard shows per-flow level data, given a particular source and destination organization. The table shows the volume, rate, retransmits, round trip time (RTT) and duration (in hours, minutes and seconds) for each flow. </center>\n<center><b>Please note that retransmit and RTT information is only available for archive flow data, and a maximum of 10,000 results are shown in the tables.</b></center>\n<center>All times are displayed in browser local time.</center>\n",
```

Comment:
This is the per country dashboard but your text includes organization details, should be changed to be more country specific?